qpc/cred/utils.py

```diff
@@ -45,6 +45,12 @@ def get_password(args, req_payload, add_none=True):
         req_payload['become_password'] = pass_prompt or None
     elif add_none:
         req_payload['become_password'] = None
+    if "token " in args and args.token:
+        print(_(messages.OPENSHIFT_TOKEN))
+        token_prompt = getpass()
+        req_payload["auth_token"] = token_prompt or None
```

Comment:
Instead of allowing None or "", check, raise an error.

Reply:
I wrote a function that will validate the pass_prompt and raise an error if empty or none.

Reply:
its technically impossible to have `token_prompt` "falsy" here. you can drop the "or" part.